gamemodes/terrortown/entities/entities/ttt_base_placeable.lua

```diff
@@ -24,6 +28,11 @@ function ENT:Initialize()
 
     if SERVER then
         self:PrecacheGibs()
+
+        local mvObject = self:AddMarkerVision("ttt_base_placeable_owner")
+        mvObject:SetOwner(self:GetOriginator())
```

Comment:
not a fan that the owner is hard set to the originator here, the owner can also be a role or a team

Reply:
For what is the owner relevant anyways? I just hardcoded it as I saw no benefit to it being customizable. Normally you place it you own it.

Reply:
The heroes crystal, that also uses this base, has the owner set to ROLE_SUPERVILLAIN. As any supervillain can see it, but a supervillain changing their role, will lose access to the marker vision